includes/class-amp-story-post-type.php

```diff
@@ -219,4 +214,21 @@ public static function filter_template_include( $template ) {
 		}
 		return $template;
 	}
+
+	/**
+	 * Add CSS to the frontend for AMP Stories.
+	 *
+	 * @see /assets/css/amp-story-frontend.css
+	 */
+	public function add_frontend_css() {
+		$post = get_post();
+		if ( 'amp_story' !== $post->post_type ) {
+			return;
+		}
+
+		$css_src      = AMP__DIR__ . '/assets/css/amp-story-frontend.css';
+		$css_contents = file_get_contents( $css_src );
+
+		wp_add_inline_style( 'wp-block-library', $css_contents );
```

Comment:
Could we also add the CSS for the editor? So that the editor would see the change there, too?

Reply:
Done, had to make some changes in the JS too for this to work.